Src/AutoFixture/TypeFilter.cs

```diff
@@ -1,5 +1,10 @@
 ﻿using System;
 using Ploeh.AutoFixture.Kernel;
+#if NETFULL
```

Comment:
Would it be better to control this with something a little more granular than `NETFULL`?